Error while building my own image

Hello, I used `docker build -t kaldi-gstreamer-server:1.0 https://github.com/jcsilva/docker-kaldi-gstreamer-server.git` to build my own image. And after the step of cloning kaldi, the buildin process break in the make. Here are a snapshot of my terminal:

  ---> Running in b43424061057
**Cloning into 'kaldi'...**
extras/check_dependencies.sh
extras/check_dependencies.sh: sox is not installed.
extras/check_dependencies.sh: we recommend that you run (our best guess):
 sudo apt-get install  sox
**make: *** [check_required_programs] Error 1**
Makefile:31: recipe for target 'check_required_programs' failed
The command '/bin/sh -c git clone https://github.com/kaldi-asr/kaldi &&     cd /opt/kaldi/tools &&     make &&     ./install_portaudio.sh &&     cd /opt/kaldi/src && ./configure --shared &&     sed -i '/-g # -O0 -DKALDI_PARANOID/c\-O3 -DNDEBUG' kaldi.mk &&     make depend && make &&     cd /opt/kaldi/src/online && make depend && make &&     cd /opt/kaldi/src/gst-plugin && make depend && make &&     cd /opt &&     git clone https://github.com/alumae/gst-kaldi-nnet2-online.git &&     cd /opt/gst-kaldi-nnet2-online/src &&     sed -i '/KALDI_ROOT?=\/home\/tanel\/tools\/kaldi-trunk/c\KALDI_ROOT?=\/opt\/kaldi' Makefile &&     make depend && make &&     rm -rf /opt/gst-kaldi-nnet2-online/.git/ &&     find /opt/gst-kaldi-nnet2-online/src/ -type f -not -name '*.so' -delete &&     rm -rf /opt/kaldi/.git &&     rm -rf /opt/kaldi/egs/ /opt/kaldi/windows/ /opt/kaldi/misc/ &&     find /opt/kaldi/src/ -type f -not -name '*.so' -delete &&     find /opt/kaldi/tools/ -type f \( -not -name '*.so' -and -not -name '*.so*' \) -delete &&     cd /opt && git clone https://github.com/alumae/kaldi-gstreamer-server.git &&     rm -rf /opt/kaldi-gstreamer-server/.git/ &&     rm -rf /opt/kaldi-gstreamer-server/test/' returned a non-zero code: 2

--- a/Dockerfile
+++ b/Dockerfile
@@ -25,6 +25,9 @@ RUN apt-get update && apt-get install -y  \
     python-gi \
     subversion \
     wget \
+    build-essential \
+    python-dev \
+    sox \
     zlib1g-dev && \
     apt-get clean autoclean && \
     apt-get autoremove -y && \

